tools/inference.py

```diff
@@ -60,10 +57,10 @@ def infer() -> None:
     extension = os.path.splitext(args.weight_path)[-1]
     inference: Inferencer
     if extension in (".ckpt"):
-        inference = TorchInferencer(config=config, path=args.weight_path)
```

Comment:
following the above argument, maybe we could do `from anomalib.deploy.inference TorchInferencer` here?